aries/aries_vcx/Cargo.toml

```diff
@@ -73,7 +73,7 @@ url = { version = "2.3", features = ["serde"] }
 backtrace = { optional = true, version = "0.3" }
 
 [dev-dependencies]
-test_utils = { path = "../misc/test_utils" }
+test_utils = { path = "../misc/test_utils", features = ["vdrtools_wallet"] }
```

Comment:
Why fix the features for `test_utils` instead of propagating the ones set in this crate?

Reply:
Removed.